test/imports-test.ts

```diff
@@ -388,11 +388,53 @@ var _moduleName = require('moduleName');
       `"use strict";
       var _mymodule = require('my-module');
       
-      (0, _mymodule.bar)();
+      _mymodule.bar.call(void 0, );
     `,
     );
   });
 
+  it("never breaks weird call expressions", () => {
```

Comment:
Can you add a new test or an extend one of these tests to include the `new` case?

Reply:
👍